.travis.yml

```diff
@@ -17,13 +17,16 @@ matrix:
         - php: 7.0
         - php: 7.1
         - php: 7.2
+        - php: 7.3
 
           # Test LTS version we support
         - php: 7.2
           env: DEPENDENCIES="symfony/lts:v3"
 
         - php: 7.2
           env: DEPENDENCIES="beta"
+        - php: 7.3
+          env: DEPENDENCIES="beta"
```

Comment:
Instead of adding this as a separate build, we should just change the LTS and beta builds to use PHP 7.3 instead of 7.2.